bazel/repository_locations.bzl

```diff
@@ -425,8 +425,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_url = "https://luajit.org",
         # The last release version, 2.1.0-beta3 has a number of CVEs filed
         # against it. These may not impact correct non-malicious Lua code, but for prudence we bump.
```

Comment:
this comment seems out of date - perhaps we can change to something like "LuaJIT only provides rolling releases"